source/draw/RasterPacked.ooc

```diff
@@ -51,7 +58,9 @@ RasterPackedCanvas: abstract class extends RasterCanvas {
 			sourceRow := (sourceHeight * row) / resultHeight + sourceStartRow
 			for (column in 0 .. resultWidth) {
 				sourceColumn := (sourceWidth * column) / resultWidth + sourceStartColumn
-				resultBuffer[(column + resultStartColumn) + resultStride * (row + resultStartRow)] = sourceBuffer[sourceColumn + sourceStride * sourceRow]
+				(resultColumnTransformed, resultRowTransformed) := This _transformCoordinates(column + resultStartColumn, row + resultStartRow, target width, target height, target coordinateSystem)
```

Comment:
No it was a stupid idea forget it. Probably because I didn't have a breakfast today :/